csrc/anemone_mempool.c

```diff
@@ -1,11 +1,12 @@
 #include "anemone_mempool.h"
 #include <string.h>
+#include <malloc.h>
 
-// Allocate a new block with given predecessor 
+// Allocate a new block with given predecessor
 ANEMONE_STATIC
 anemone_block_t * anemone_block_create (anemone_block_t *prev, size_t num_bytes)
 {
-  void     *ptr = malloc (num_bytes);
+  void     *ptr = memalign(ANEMONE_ALIGNMENT, num_bytes);
```

Comment:
I think `malloc` will always do 8 byte alignments on x86_64, but we should keep this in case we ever need to test and `ANEMONE_ALIGNMENT` value of say `16`.